src/components/Recipe/Form/useInputState.tsx

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+function useInputState(initialVal: any){
```

Comment:
Try to never use `any`, it kills the purpose of using typescript. In our apps you won't be allowed to use it.